pkg/cmd/experimental/buildchain/buildchain.go

```diff
@@ -23,431 +21,166 @@ const (
 	buildChainLong = `
 Output the inputs and dependencies of your builds
 
-Supported output formats are json, dot, and ast. The default is set to json.
+Supported formats are dot and plain, a human-readable output. The default is set to dot.
 Tag and namespace are optional and if they are not specified, 'latest' and the 
 default namespace will be used respectively.`
 
 	buildChainExample = `  // Build dependency tree for the specified image stream and tag
-  $ openshift ex build-chain [image-stream]:[tag]
+  $ %[1]s build-chain [image-stream]:[tag]
 
   // Build dependency trees for all tags in the specified image stream
-  $ openshift ex build-chain [image-stream] --all-tags
+  $ %[1]s build-chain [image-stream] --all-tags
 
-  // Build the dependency tree using tag 'latest' in 'testing' namespace
-  $ openshift ex build-chain [image-stream] -n testing
-
-  // Build the dependency tree and output it in DOT syntax
-  $ openshift ex build-chain [image-stream] -o dot
+  // Build the dependency tree and output it in a human readable output
+  $ %[1]s build-chain [image-stream] -o plain
 
   // Build dependency trees for all image streams in the current namespace
-  $ openshift ex build-chain
+  $ %[1]s build-chain
 
   // Build dependency trees for all image streams across all namespaces
-  $ openshift ex build-chain --all`
+  $ %[1]s build-chain --all`
 )
 
-// Node is a representation of an image stream inside a tree
-type Node struct {
-	FullName string   `json:"fullname"`
-	Tags     []string `json:"tags,omitempty"`
-	Edges    []*Edge  `json:"edges,omitempty"`
-	Children []*Node  `json:"children,omitempty"`
-}
-
-// String helps in dumping a tree in AST format
-func (root *Node) String() string {
-	tree := ""
-	tree += root.FullName
+// BuildChainOptions contains all the options needed for build-chain
+type BuildChainOptions struct {
```

Comment:
No, it's not. I first want to get the stuff in dot together and then I'll fix these.